google-api-client/pom.xml

```diff
@@ -36,20 +36,42 @@
         <configuration>
           <archive>
             <manifest>
-              <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
             </manifest>
+             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
           </archive>
         </configuration>
         <executions>
           <execution>
             <id>jar</id>
-            <phase>compile</phase>
+            <phase>package</phase>
             <goals>
               <goal>jar</goal>
             </goals>
           </execution>
         </executions>
       </plugin>
+      <plugin>   
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>2.5.4</version>
+        <executions>
+          <execution>
+            <id>bundle-manifest</id>
+            <phase>process-classes</phase>
+            <goals>    
+              <goal>manifest</goal>
+            </goals>   
+          </execution>
+        </executions>
+        <configuration>
+          <instructions>
+            <Bundle-DocURL>https://developers.google.com/api-client-library/java/</Bundle-DocURL>
+            <Bundle-Description>The Google API Client Library for Java provides functionality common to all Google APIs; for example HTTP transport, error handling, authentication, JSON parsing, media download/upload, and batching.</Bundle-Description>
+            <Bundle-SymbolicName>com.google.api.client.googleapis</Bundle-SymbolicName>
```

Comment:
It looks like you're using a different set of instructions from https://github.com/google/google-oauth-java-client/commit/57fd88b34bea69005c0a3bf0ad547a7a3acb2005 - what are the reasons?